swift-ticketing/src/swift_ticketing/db/booking.clj

```diff
@@ -1,25 +1,34 @@
 (ns swift-ticketing.db.booking
-  (:require [honey.sql :as sql]))
+  (:require [honey.sql :as sql]
+            [swift-ticketing.db.query :refer [run-query]]))
 
 (defonce INPROCESS "InProcess")
 (defonce CONFIRMED "Confirmed")
 (defonce PAYMENTPENDING "PaymentPending")
 (defonce REJECTED "Rejected")
 (defonce CANCELED "Canceled")
 
-(defn insert-booking [uid booking-id]
+(defn insert-booking-query [uid booking-id]
```

Comment:
Does this need to be public? (or even exist separately from the `insert-booking` function?